pkg/cmd/login/login.go

```diff
@@ -202,9 +202,9 @@ func runLogin(opts *Options) (err error) {
 	opts.Logger.Info("")
 
 	if !ok {
-		opts.Logger.Info(icon.Success(), opts.localizer.MustLocalize("login.log.info.loginSuccessNoUsername"))
+		opts.Logger.Info(icon.SuccessPrefix(), opts.localizer.MustLocalize("login.log.info.loginSuccessNoUsername"))
```

Comment:
Just idea:  `opts.Logger.Success`

Reply:
Yeah, that might make things nicer actually, though I do like keeping the Logger lean and focused on log-levels only.

Reply:
our logger is not logger per se. It is an stderr printer. Having more high-level methods will be good idea for future issue.

Reply:
Yeah, but the stderr printer is an implementation of a higher-level Logger interface, which could be implemented in different contexts. Though I do not see it being used in ways other than to stderr